### DISCUSSION TITLE
Prevent public access to the admin App login page?

### DISCUSSION THREAD

**Discussion by altezza04** [directus/directus#6571] [Archive]
How to prevent the admin App login page access by the public or may be a way to change the default link?

currently I have https://localhost/ will directly link to the login page /admin/login, anyway to change that?

**Comment by rijkvanzanten** [comment]
See ROOT_REDIRECT https://docs.directus.io/reference/environment-variables/#general

**Reply by altezza04** [reply]
thanks for the quick reply. I also want to ask how to change "/admin" path to something else ?

**Reply by rijkvanzanten** [reply]
That's not currently possible
